src/viewers/PictureInPictureViewer.js

```diff
@@ -52,7 +54,9 @@ const PictureInPictureViewer = props => {
     views.push(overviewView);
     layerProps.push(layerConfig);
   }
-  return loader ? <VivViewer layerProps={layerProps} views={views} /> : null;
```

Comment:
in these situations I somewhat prefer:

```javascript
if (!loader) return null;
return <VivViewer {...} />
```

Maybe just a stylistic preference, but it makes it clear to me what conditions need to be fulfilled before rendering component, without parsing a ternary.